frontend/public/kubevirt/components/vm/menu-actions.jsx

```diff
@@ -65,7 +65,7 @@ const menuActionStart = (kind, vm, actionArgs) => {
 
 const menuActionRestart = (kind, vm, actionArgs) => {
   return {
-    hidden: !isVmRunning(vm) || isImporting(vm, actionArgs),
+    hidden: !(actionArgs[VirtualMachineInstanceModel.kind] && isVmRunning(vm)),
```

Comment:
In the new version we removed the check for `isImporting` is this intentional ?

Reply:
that is intentional. I dont see reason to have that check - we can restart the machine when its running & VMI exists I dont think we need any other checks

Reply:
Thanks :+1:

Reply:
I vote for keeping the check here. Otherwise it would allow for some other mistake to propagate. 

For example:
During v2v migration, it is possible to start VM from CLI. This will make VMI running and importing at the same time. The only valid action here is to stop the vm and hope for the best (that the migration suceeds). So I think the UI should represent only the valid options.

In that regard, I think we should enhance the stop action and show that any time the VM is running.

Reply:
Ok, you are right. I've improved start/stop action too, please check